spec/worker-blocks-positions-spec.coffee

```diff
@@ -64,11 +64,13 @@ describe 'worker when the scroll mode is activated', ->
 
     runs =>
       expect(Object.keys @blockPositions).toHaveLength 5
-      expect(@blockPositions[1]).toBe '__asciidoctor-preview-420__'
+      expect(@blockPositions[1]).toBe '__asciidoctor-preview-1554__'
       expect(@blockPositions[3]).toBe '_first_section'
-      expect(@blockPositions[6]).toBe '__asciidoctor-preview-468__'
-      expect(@blockPositions[7]).toBe '__asciidoctor-preview-492__'
-      expect(@blockPositions[8]).toBe '__asciidoctor-preview-512__'
+      # line number on nested list is wrong
+      # maybe related to https://github.com/asciidoctor/asciidoctor/pull/2547
```

Comment:
@mojavelinux probably related to https://github.com/asciidoctor/asciidoctor/pull/2547 right ?

Reply:
That's correct.